### ISSUE TITLE
Cursor visibility

### ISSUE THREAD

**Issue by kyrieb-ekat** [DDMAL/Pixel.js#279]
While highlighting, selecting, or in brush mode, particularly while very zoomed in, cursor can be difficult to spot. Possible to change cursor colour to be more visible even while brush size is very small?

**Comment by JoyfulGen** [comment]
I bumped up the priority of this issue, because the cursor is almost completely invisible against the new Aquitanian manuscript! It's still totally possible to use it, but looking for the cursor is tiring for the eyes and very time-consuming. We'd be really grateful for a fix!

Here's a demo, where you can see that once I drag my cursor over the image it becomes almost impossible to track:
https://github.com/user-attachments/assets/3012a810-2aad-45e9-9385-c452ca8497ee

**Comment by fujinaga** [comment]
@yinanazhou We need to keep the pointer size constant regardless of the Zoom level. This is a highly needed fix and should be fairly easy to fix with your expertise ;-).

**Comment by kyrieb-ekat** [comment]
Here's the workflow containing pixel, and a small image strip you can assign as its resource to test this @yinanazhou 
[pixel only.json](https://github.com/user-attachments/files/18127899/pixel.only.json)
![MS73 288 - 4of4](https://github.com/user-attachments/assets/d7296860-3c6c-4c63-a1da-5a6a7e956c78)

**Comment by yinanazhou** [comment]
Can this be closed?

**Comment by JoyfulGen** [comment]
Yes! The brush colour wheel is very handy and is such a relief for the eyes. Thank you!
